packages/cli/src/commands/init/editTemplate.ts

```diff
@@ -101,49 +101,33 @@ async function processDotfiles(filePath: string) {
   await renameFile(filePath, `_${dotfile}`, `.${dotfile}`);
 }
 
-function getPackageNameDetails(packageName: string) {
-  const cleanPackageName = packageName.replace(/[^\p{L}\p{N}.]+/gu, '');
```

Comment:
i think we should still sanitize package name with this regex, to prevent failing to process the template when someone type e.g. `--package-name "c[om.package.name"